options.go

```diff
@@ -178,6 +187,16 @@ func WithHTTPClient(client *http.Client) Option {
 func WithAdminAPIEndpoint(url string) Option {
 	return func(d *dialerConfig) {
 		d.sqladminOpts = append(d.sqladminOpts, apiopt.WithEndpoint(url))
+		d.serviceUniverseDomain = strings.SplitN(url, ".", 2)[1]
+	}
+}
+
+// WithUniverseDomain configures the underlying SQL Admin API client to use
+// the provided universe domain.
```

Comment:
Should we add something about Trusted Private Cloud and link to a docs page for more information?

Reply:
added additional details to comment... "Enables Trusted Partner Cloud (TPC)."